lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -112,7 +131,7 @@ def get_results(self, name_list: List[str]) -> JSON:
             ("drb", {"end": "1970-09-01"}, []),
         ],
     )
-    def test_dataset_list(self, query_as, login, query, results):
+    def test_dataset_list(self, query_as, login, query, results, server_config):
```

Comment:
You no longer need the `server_config` fixture here, right?  Ditto line 148.

Reply:
Yes I do, to construct the next url